examples/advanced/custom-templates/README.md

```diff
@@ -0,0 +1,73 @@
+## Custom templates
+
+This example shows how to use "custom" templates to generate design tokens files with custom formats. This is useful when you need to distribute your design tokens and integrate them with custom pipelines or scripts, that expect specific formats, or if you have very specific needs that are not covered out-of-the-box by Style Dictionary.
+
+**Notice**: before starting to dig into all the possible customisations that you can have, try the default settings offered by the library, look at the output files, and see if they can suit your needs. Probably they will do. If they don’t, think how you want the output files generated, and see which one of the API methods you can use for that specific scope.
+
+#### Running the example
```

Comment:
I see what you mean. Mmm, on one side it would be good to have it at the "examples" root level, on the other if someone doesn't read the main Readme they could miss it. I'll think about a solution.